Formal dresses for heads!

HTMLSystem opened the pull request:
<!-- If you are new to the Delta-V repository, please read the [Contributing Guidelines](https://github.com/DeltaV-Station/Delta-v/blob/master/CONTRIBUTING.md) -->

## About the PR
<!-- What did you change? -->
Added formal dresses for all the heads that have a formal suit, but not a formal dress. Those heads are the LO, HoP, and CJ. Also added inhand sprites for the HoP's formal suit and added it to the uniform printer.

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Because I think it's silly and dumb that there are only suits, but not dresses. Now anyone feminine can be formal as well!

## Technical details
<!-- Summary of code changes for easier review. -->
Added new folders and sprites for all dresses, added them to the loadouts, and added them to the uniform printer.

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc).
Small fixes/refactors are exempt. -->
![image](https://github.com/user-attachments/assets/93cb34b4-5dd0-4509-86fc-52d1db0274ee)
![image](https://github.com/user-attachments/assets/1cce4bde-0000-4e49-8e84-8415570c5dfa) ![image](https://github.com/user-attachments/assets/a814c41e-a829-41cc-be67-bfa02822a522) ![image](https://github.com/user-attachments/assets/2086d0d6-ad7c-4478-a4e2-076612a966e8)


## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelogs must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->
:cl:
- add: Added formal dresses for the LO, HoP, and CJ! Find them in their dressers, lockers, loadouts, and uniform printer!
- add: Added the HoP's formal suit to the uniform printer!

HTMLSystem committed:
Add LO's formal dress

Added it ingame, into loadouts, to the uniform printer, and to the dresser

HTMLSystem committed:
Adds cj formal dress + fixes some bugs

Adds the cj formal dress into existance, to the uniform printer, and to the cj locker. Also fixes the LO's inhand sprites and it's uniform printer recipe.

HTMLSystem committed:
Added HoP's formal dress + other things

Added HoP's dress into existance, to the uniform printer, to the loadout, and to the HoP's locker. Also adds the HoP's formal suit to the uniform printer, adds inhand sprites for the suit, and fixes the description of all the dresses to match their suit conterparts.

HTMLSystem committed:
Touch up

Corrected the id names of the dresses because it bothered me that it didn't follow the format.

HTMLSystem committed:
Merge branch 'master' into Formal-dresses

Signed-off-by: HTML/Crystal <152909599+HTMLSystem@users.noreply.github.com>

HTMLSystem committed:
Guh - Bug fixes

HTMLSystem committed:
Merge branch 'Formal-dresses' of https://github.com/HTMLSystem/Delta-v into Formal-dresses

HTMLSystem commented:
Ready to merge now !

Lyndomen commented on Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpskirts.yml:
```diff
@@ -33,6 +33,28 @@
+  parent: ClothingUniformSkirtBase
+  id: ClothingUniformJumpskirtHoPFormal
+  name: head of personnel's formal dress
+  description: A well-made dress with gold buttons and a red trim. Might as well look sharp when you blow up.
```
can you change blow up to something like service disaster related

HTMLSystem replied:
Sure thing! Would you like me to change the description on the suit as well?

Lyndomen commented on Resources/Textures/DeltaV/Clothing/Uniforms/Jumpskirt/cjformaldress.rsi/meta.json:
```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC0-1.0",
+  "copyright": "Made by HTMLSystem, edited from https://github.com/ParadiseSS13/Paradise/",
```
is this accurate? it sounds like it could read "made by paradisess13, edited by HTMLSystem" unless im major wrong

deltanedas replied:
true also paradise assets are CC-BY-SA-3.0 i think so this cant be CC0-1.0, also use a permalink to the dmi its based on

HTMLSystem replied:
I basically just copy pasted what the HoP formal suit says in it's copyright thing, but yeah I can edit it.

HTMLSystem replied:
I lied, appearently I just fucked up the copy/pasting somehow

HTMLSystem committed:
Fix ops part 2

Edited the HoP's formal outfits to have a more service-orinted disaster mentioned and fixed the licensing on the HoP's formal dress. Idk how I fucked that one up.

HTMLSystem commented:
Tis ready again

Commit:
merge master

Commit:
untroll

Commit:
remove deltav comment in deltav file

HTMLSystem commented:
Thank you for fixing all the open prs Delta that's pretty awesome :)

deltanedas commented:
christmas miracle
